src/Utilities/ExcelImporterUtils.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+
+namespace ExcelMapper.Utilities
+{
+    public static class ExcelImporterUtils
+    {
+#if NETSTANDARD2_0
+        public static List<ExcelClassMap> RegisterMapperClassesByNamespace(this ExcelImporter importer,
+            string namespacestr)
+        {
+            var classes = Assembly.GetExecutingAssembly()
+                .GetTypes().AsParallel()
+                .Where(t => t.IsClass && t.Namespace == namespacestr);
+
+            var objects = classes.AsParallel().Select(Activator.CreateInstance)
+                .AsParallel().OfType<ExcelClassMap>().ToList();
```

Comment:
Are all these calls to `AsParallel` necessary? Surely only one is necessary? I'm not sure

Reply:
Don't know for sure, but removed it anyway